save-demo/src/main/kotlin/com/saveourtool/save/demo/entity/Demo.kt

```diff
@@ -46,7 +46,10 @@ class Demo(
         }
     }
 
-    private fun projectCoordinates() = ProjectCoordinates(
```

Comment:
could it be an class field, instead of manual crating a new instance any time?

Reply:
No, it couldn't as `Demo` is an `entity` and the only solution is to create a table for `ProjectCoordinates` which is weird

Reply:
Oh I guess I got your point here, but not sure if we really want to do it in this PR. Probably should be done in one of next PRs with `Demo` refactoring (adding a new column)